fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -2658,7 +2658,7 @@ public void clearExpiredJobs() {
             LOG.warn("task manager clean expire tasks failed", t);
         }
         try {
-            taskManager.removeExpiredTaskRuns();
+            taskManager.removeExpiredTaskRuns(false);
```

Comment:
`labelCleaner` is running in both leader and follower FE.
`archive history` must be done if the FE is leader.